yt_dlp/extractor/chzzk.py

```diff
@@ -0,0 +1,121 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    traverse_obj,
+    parse_iso8601,
+)
+
+
+class CHZZKLiveIE(InfoExtractor):
+    IE_NAME = 'chzzk:live'
+    _VALID_URL = r'https?://chzzk\.naver\.com/live/(?P<id>[^/#?]+)'
+    _TESTS = [{
+        'url': 'https://chzzk.naver.com/live/c68b8ef525fb3d2fa146344d84991753',
+        'only_matching': True,
```

Comment:
Even if users are not constantly live, I would still add a test. Maybe skip it by default.